scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/edf/EDFPolicyExtended.java

```diff
@@ -63,6 +64,8 @@
 
     private static final Date MAXIMUM_DATE = new Date(Long.MAX_VALUE);
 
+    private static final Logger LOGGER = Logger.getLogger(EDFPolicyExtended.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOGGER" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)